Tests/regressionTests.js

```diff
@@ -0,0 +1,282 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
+var fashionCategory = require ("../Pages/fashionCategoryPage");
+var itemPage = require ("../Pages/itemPage");
+var registerPage = require ("../Pages/registerPage");
+var aboutUs = require("../Pages/aboutUs");
+var termsandconditions = require("../Pages/termsAndConditions");
+var privacyAndPolicy = require ("../Pages/privacyAndPolicy");
+var dataJS = require("../Pages/data");
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+
+
+describe ("002: Registration", function (){
+    beforeEach(() => {
+        homepage.open(dataJS.homepageLink)
+        .then(() => homepage.clickOnCreateAnAccont())
+    })
+    
+    var random1 = Math.round(Math.random()*100);
+    var random2 = Math.round(Math.random()*100);
+
+    it ("001: User is able to create account with valid credentials", function (){
+        var firstName="Tester";
+        var lastName="123";
+        var email = `gigedov${random1}${random2}@getnada.com`;
+        var password ="testing1";
+        registerPage.enterFirstName(firstName)
+        .then(() => registerPage.enterLastName(lastName))
+        .then(() => registerPage.enterEmail(email))
+        .then(() => registerPage.enterPassword(password))
+        .then(() => registerPage.enterConfirmPassword(password))
+        .then(() => registerPage.clickOnRegisterButton())
+        .then(() => browser.wait(dataJS.EC.presenceOf($(dataJS.statusSuccessLocator)), 5000))
+        .then(() => registerPage.getRegistrationMessage())
+        .then((data) => expect(data).toBe(dataJS.successfullRegistrationMessage))
+    })
+
+    it ("002: User isn't able to create account with invalid credentials", function (){
+        registerPage.enterFirstName(dataJS.dataWithEmptySpaces)
+        .then(() => registerPage.enterLastName(dataJS.dataWithEmptySpaces))
+        .then(() => registerPage.enterEmail("gigedov@getnada"))
+        .then(() => registerPage.enterPassword(dataJS.dataWithEmptySpaces))
+        .then(() => registerPage.enterConfirmPassword(dataJS.dataWithEmptySpaces))
+        .then(() => registerPage.clickOnRegisterButton())
+        .then(() => browser.wait(dataJS.EC.presenceOf($(".status-error")), 5000))
```

Comment:
Every click, wait and expect should be removed to page object

Reply:
I removed this, and added them in methods